contrib/libbsd/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "libbsd"
```

Comment:
what is this actually for?

Reply:
@nekopsykose working on porting https://gameoftrees.org/ but it's still wip